mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MessageMapperTest.java

```diff
@@ -265,7 +271,9 @@ public void messagesRetrievedUsingFetchTypeHeaderShouldHaveHeaderDataLoaded() th
         MessageMapper.FetchType fetchType = MessageMapper.FetchType.Headers;
         Iterator<MailboxMessage> retrievedMessageIterator = messageMapper.findInMailbox(benwaInboxMailbox, MessageRange.one(message1.getUid()), fetchType, LIMIT);
         assertThat(retrievedMessageIterator.next()).isEqualTo(message1, fetchType);
-        assertThat(retrievedMessageIterator).isEmpty();
+        assertThat(retrievedMessageIterator)
+            .toIterable()
```

Comment:
why have you done that?

Reply:
Seems like it's the new syntax, had to explicitly transform it to an iterable... Or it just doesn't compile